apps/client/src/locales/messages.it.xlf

```diff
@@ -4,6 +4,7 @@
     <body>
       <trans-unit id="e19fcf996343543e13789f17b550ab0c08124b5c" datatype="html">
         <source>Create Account</source>
+		<target state="translated">Crea un account</target>
```

Comment:
Here also (tab vs. spaces), maybe the conflicts go away with that change...

Reply:
Done